docs/containers-conf.md

```diff
@@ -17,65 +18,80 @@ data:
   HA_CONNECTION_TIMEOUT: MTAwMA==
   OK_IF_DONOR: MQ==
   HA_SERVER_OPTIONS: Y2hlY2sgaW50ZXIgMzAwMDAgcmlzZSAxIGZhbGwgNSB3ZWlnaHQgMQ==
+  PEER_LIST_SRV_PROTOCOL: dGNw
 ```
 
 !!! note
 
-     <a name="haproxy-options"> Variables used in this example have the following effect:
-     
-     * `HA_CONNECTION_TIMEOUT` allows to set custom timeout for health checks done by HAProxy (it repeatedly executes a simple status query on XtraDB Cluster instances). The default 10 seconds timeout is good for most workloads, but increase should be helpful in case of unstable Kubernetes network or soft lockups happening on Kubernetes nodes.
-     * `OK_IF_DONOR` allows application connections to XtraDB Cluster donors. The backup is running on the donor node, and SQL queries combined with it could run slower than usual. Enable the option to grant application access when there is only one XtraDB Cluster node alive, and a second XtraDB Cluster node is joining the cluster via SST.
-     * `HA_SERVER_OPTIONS` allows to set the [custom options :octicons-link-external-16:](https://docs.haproxy.org/2.6/configuration.html#5) for the server in the HAProxy configuration file. You can start with the default `check inter 30000 rise 1 fall 5 weight 1` set, and add required options [referenced in the upstream documentation :octicons-link-external-16:](https://docs.haproxy.org/2.6/configuration.html#5.2).
+    The variables in this example have the following effects:
+    
+    * `HA_CONNECTION_TIMEOUT` sets a custom timeout for HAProxy health checks. HAProxy repeatedly executes status queries on XtraDB Cluster instances. The default 10-second timeout works for most workloads, but you may need to increase it for unstable Kubernetes networks or when soft lockups occur on Kubernetes nodes.
+    
+    * `OK_IF_DONOR` allows application connections to XtraDB Cluster donor nodes. Donor nodes run backups, which can slow down SQL queries. Enable this option when only one XtraDB Cluster node is available and a second node is joining the cluster via SST.
+    
+    * `HA_SERVER_OPTIONS` sets [custom options :octicons-link-external-16:](https://docs.haproxy.org/2.6/configuration.html#5) for servers in the HAProxy configuration file. The default is `check inter 30000 rise 1 fall 5 weight 1`. You can add additional options [referenced in the HAProxy documentation :octicons-link-external-16:](https://docs.haproxy.org/2.6/configuration.html#5.2).
+    
+    * `PEER_LIST_SRV_PROTOCOL` enables the Operator to use TCP protocol when performing peer-list SRV lookups. This may be required in environments where DNS SRV lookups need to use TCP instead of UDP, such as when dealing with large DNS responses or when UDP DNS queries are blocked by network policies.
```

Comment:
this sounds like the env var is only for enabling TCP but actually it's to configure protocol for lookups. you can set it to tcp or udp